network-api/networkapi/wagtailpages/pagemodels/customblocks/recent_blog_entries.py

```diff
@@ -72,10 +66,14 @@ def get_context(self, value, parent_context=None):
         if category:
             type = "category"
             query = slugify(category)
-            BlogPageCategory = apps.get_model('wagtailpages.BlogPageCategory')
-            category_object = BlogPageCategory.objects.get(name=category)
-            blogpage.extract_category_information(category_object.slug)
-            entries = blogpage.get_entries(context)
+            try:
```

Comment:
I can't recall how `tag` is set up. Curious why only `category` has to go through try-except and `tag` deosn't?

Reply:
tag gets try/excepted over in `filter_entries_for_tag`, called as part of `get_entries`

Reply:
https://github.com/mozilla/foundation.mozilla.org/blob/master/network-api/networkapi/wagtailpages/pagemodels/index.py#L113-L120